spec/dummy/app/controllers/rademade_admin/posts_controller.rb

```diff
@@ -32,7 +35,7 @@ class RademadeAdmin::PostsController < RademadeAdmin::ModelController
     end
     csv do
       headline
-      status handler: Proc.new {|post| 'Status'}
+      status :handler => Proc.new { 'Status' }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.